views/home.tmpl

```diff
@@ -7,7 +7,7 @@
     {{ template "navbar" . }}
     <div class="container main" data-controller="time blocklist homepage">
         <div class="row">
-            <div class="col-md-15 p-0">
+            <div class="d-flex flex-column justify-content-between col-md-15 p-0">
```

Comment:
This doesn't quit work with exchange data. Turn on exchange monitoring in your configuration to see that. 

![image](https://user-images.githubusercontent.com/6109680/60886136-c8999780-a216-11e9-8eaf-f6b6d9182c59.png)

You'll need to be a little more strategic with the flexbox. I'd imagine you'll need to set flex on the children elements (the "cards") too.

Also, you seem to be missing your treasury balance, which may be problematic. Are you seeing any errors?

Reply:
I have replicated that display bug..looking into it.

What do you mean by my treasury balance? Is it the 626,369 DCR under the Distribution section?

Reply:
Sorry I missed this. I meant that the `Treasury` was missing from your original screenshots.